.github/workflows/update-aaguids.yml

```
@@ -0,0 +1,36 @@
+name: Update AAGUIDs
+
+on:
+  schedule:
+    - cron: "0 0 * * 0" # Runs every Sunday at midnight
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+jobs:
+  update-aaguids:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install jq
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y jq
+
+      - name: Fetch JSON data
+        run: |
+          curl -o data.json https://raw.githubusercontent.com/passkeydeveloper/passkey-authenticator-aaguids/refs/heads/main/combined_aaguid.json
+
+      - name: Process JSON data
+        run: |
+          mkdir -p backend/resources
+          jq 'map_values(del(.icon_light, .icon_dark))' data.json > backend/resources/aaguids.json
+
```

Add unit tests. Because GitHub doesn't run Actions in response to a commit triggered by another action, this should be done before the commit


```suggestion

      - uses: actions/setup-go@v5
        with:
          go-version-file: 'backend/go.mod'
          cache-dependency-path: 'backend/go.sum'
      - name: Install dependencies
        working-directory: backend
        run: |
          go get ./...
      - name: Run backend unit tests
        working-directory: backend
        run: |
          go test -v ./internal/utils/...

```

---

I dont think we need to run the unit tests here. What purpose does it have? i plan on either adding a check to see if its null, or i am waiting to hear from elias about making our own fork of it.

---

I think have a more robust check would be better, or having our own list.

---

This is no longer a issue, i createds a custom repo with the same logic as the aaguid repo but this way we can make sure the list does not go away.